app/2.0/start/quick-tour.md

```diff
@@ -33,6 +33,12 @@ of the samples in an interactive sandbox.
 
 Tap the buttons following each feature to learn more.
 
+**The Plunker samples won't work in Internet Explorer 11.** These code samples need to be transpiled 
+from ES6 to ES5 to work in Internet Explorer; at the moment we don't have a way to do this. The 
```

Comment:
"at the moment we don't have a way to do this" => sounds like we can't transpile ES6 or support IE.

Maybe: "This is usually done as a build step. At the moment, we don't have a way to transpile the interactive samples on Plunker."

Reply:
Much better, thanks! Will change to that.